### PR TITLE
Feature/ts 1109 add secondary page header type to text input in amino

### PR DESCRIPTION
**Pull request by Thienhuynh95**
## Linear issue

[TS-1109 : Add secondary page header type to `Text` input in Amino](https://linear.app/zonos/issue/TS-1109/add-secondary-page-header-type-to-text-input-in-amino)

## Description

This PR adds new `secondary-page-header` type to `Text` component.

## Todo

- [x] Bump version and add tag

### CODE REVIEW

**Review by goodpickle — APPROVED** [review]

File: src/components/text/Text.tsx [hunk 1]
```diff
@@ -18,12 +18,19 @@ type OtherText = (typeof textOthers)[number];
 
 export const textOptions = [
   {
-    label: 'Page Header (3XL) · 34px (2.125rem)',
+    label: 'Page Header (3XL) · 30px (1.875rem)',
```
Comment (evad1n) [inline src/components/text/Text.tsx:21]: I think this should be 32px according to the design

Comment (Thienhuynh95) [reply src/components/text/Text.tsx:21]: I just copied text straight from the design 😆 
image

Comment (Thienhuynh95) [reply src/components/text/Text.tsx:21]: https://www.figma.com/design/WnKnmG7L3Q74hqPsw4rbEE/Amino-2.0?node-id=2100-71774&m=dev

Comment (evad1n) [reply src/components/text/Text.tsx:21]: interesting 
image

Comment (Thienhuynh95) [reply src/components/text/Text.tsx:21]: Interesting 🤔 That's weird 😆

**Pull request merged**